FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
+                if name == "schedule" and self.__schedule is not None:
+                    self.__schedule.extend(item)
+                else:
+                    self.__setattr__(name,item)
             elif isinstance(item,_Unlabelable):
                 self.add_(item)
             elif isinstance(item,ProcessModifier):
                 item.apply(self)
+            elif isinstance(item,Process):
```

Comment:
I thought about that, but a SubProcess is not an instance of Process, so it should not be an issue ?